### PR TITLE
Improve Hive view docs

### PR DESCRIPTION
**Pull request by mosabua**

### CODE REVIEW

File: docs/src/main/sphinx/connector/hive.rst [hunk 1]
```diff
@@ -375,11 +381,10 @@ Property Name                                      Description
                                                    is used for write operations. The ``${USER}`` placeholder
                                                    can be used to use a different location for each user.
 
-``hive.translate-hive-views``                      Enable translation for Hive views. (experimental)            ``false``
+``hive.translate-hive-views``                      Enable translation for :ref:`Hive views <hive-views>`.       ``false``
 
-``hive.legacy-hive-view-translation``              Use the legacy algorithm to translate Hive views. You can    ``false``
-                                                   alternatively set the ``legacy_hive_view_translation``
```
Comment (mosabua) [inline docs/src/main/sphinx/connector/hive.rst:381]: This causes a misunderstanding that it is a session property .. when in fact it is a catalog session property

Comment (findepi) [reply docs/src/main/sphinx/connector/hive.rst:381]: We should list session properties directly in this table, where available.
For example we have `hive.timestamp-precision` above and i find it user-unfriendly not to mention `timestamp_precision` catalog session property next to it.

 @electrum might have  thoughts as well

Comment (mosabua) [reply docs/src/main/sphinx/connector/hive.rst:381]: I can add it back in for the current property .. doing it for all the others is a separate, much larger task

Comment (mosabua) [reply docs/src/main/sphinx/connector/hive.rst:381]: Arguably we could have another column in this kind of tables .. Catalo property, Catalog session property, Description, Default

But that should be consistent across all the docs.. we could do a mass update and convert all tables to use list-table and then add it..

**Review by electrum — APPROVED** [review]

File: docs/src/main/sphinx/connector/hive.rst [hunk 2]
```diff
@@ -375,11 +381,12 @@ Property Name                                      Description
...
-                                                   session property to ``true``.
+``hive.legacy-hive-view-translation``              Use the legacy algorithm to translate                        ``false``
+                                                   :ref:`Hive views <hive-views>`. You can use the
+                                                   ``legacy_hive_view_translation`` catalog session property
+                                                   for temporary, catalog specific use.
```
Comment (findepi) [inline docs/src/main/sphinx/connector/hive.rst:389]: > catalog specific

everything here is for catalog specific.. did you mean "session specific"?

Comment (mosabua) [reply docs/src/main/sphinx/connector/hive.rst:389]: no .. I meant catalog specific .. users seems to forget that a catalog session property is per catalog and per session. Esp when we just use the name session property and imply catalog from context it is completely missed .. this misunderstanding causes the issue btw..

**Pull request merged**